deploy/kubernetes/base/kustomization.yaml

```diff
@@ -1,7 +1,7 @@
 commonLabels:
-  app: gcp-compute-persistent-disk-csi-driver
+  k8s-app: gcp-compute-persistent-disk-csi-driver
 namespace:
-  default
+  pdcsi-serving
```

Comment:
is `gcp-compute-persistent-disk-csi-driver` too wordy? or `pd.csi.storage.gke.io` or some part thereof of either of those?. Basically just any of the names we already have for the driver instead of coming up with with yet another name

Reply:
Typing something like `k get po -n pd.csi.storage.gke.io` will be a pain, but worst case I could have an alias so not a huge deal :)

Let me know what you'd prefer the most in the end

Reply:
gce-pd-csi-driver?